tests/integration/_cli2runner.py

```diff
@@ -1,5 +1,6 @@
 """test from CLI up to runner
 """
+# pylint: disable=preferred-module  # FIXME: remove once migrated per GH-872
```

Comment:
Should we renable the check after the import so nothing else sneaks through in the future

Reply:
Not sure. But given that nothing else is blocked, it shouldn't matter now.

Reply:
@cidrblock resolve this if you agree.

Reply:
I would prefer it get added, but you are right, it doesn't matter today, but this is one of those things that I remember because I've paid the price for a single, global pylint disable in the past which let a growing file get more and more errors.

I also think it's a good pattern to get into, as an example in the code, because I was suprised to learn about this and I often wonder if others would be too.

That said, I will leave it up to you to decide and see what you do :)